app/assets/src/components/views/discovery/mapping/MapPlayground.jsx

```diff
@@ -83,102 +81,60 @@ class MapPlayground extends React.Component {
     );
 
     return (
-      <Marker key={`marker-${index}`} latitude={lat} longitude={lon}>
+      <Marker key={`marker-${index}`} latitude={lat} longitude={lng}>
         <CircleMarker
           size={markerSize}
-          onMouseOver={() =>
-            this.handleMarkerMouseOver({ lat, lon, name, pointCount })
-          }
-          onMouseOut={this.handleMarkerMouseOut}
-          onClick={() =>
-            this.openPopup({
-              lat,
-              lon,
-              name,
-              index,
-              items: markerData.items
-            })
+          onMouseEnter={() =>
+            this.handleMarkerMouseEnter({ lat, lng, name, pointCount })
           }
+          onMouseLeave={this.handleMarkerMouseLeave}
         />
       </Marker>
     );
   };
 
-  handleMarkerMouseOver = hoverInfo => {
-    const hoverTooltip = (
-      <MapPopup
-        className={cs.dataTooltipContainer}
-        anchor="top-left"
-        tipSize={0}
-        latitude={hoverInfo.lat}
-        longitude={hoverInfo.lon}
-        closeButton={false}
-        offsetLeft={15}
-        offsetTop={15}
-      >
-        <DataTooltip
-          data={[
-            { name: hoverInfo.name, data: [["Samples", hoverInfo.pointCount]] }
-          ]}
-        />
-      </MapPopup>
+  handleMarkerMouseEnter = hoverInfo => {
+    const title = `${hoverInfo.pointCount} Sample${
+      hoverInfo.pointCount > 1 ? "s" : ""
+    }`;
+    const tooltip = (
+      <MapTooltip
+        lat={hoverInfo.lat}
+        lng={hoverInfo.lng}
+        title={title}
+        body={hoverInfo.name}
+        onMouseEnter={this.handleTooltipMouseEnter}
+        onMouseLeave={this.handleMarkerMouseLeave}
+      />
     );
-    this.setState({ hoverTooltip });
-  };
-
-  handleMarkerMouseOut = () => {
-    this.setState({ hoverTooltip: null });
+    this.setState({ tooltip, tooltipShouldClose: false });
   };
 
-  openPopup = popupInfo => {
-    this.setState({
-      popups: concat(this.state.popups, popupInfo),
-      hoverTooltip: null // Replace the open tooltip
-    });
-  };
+  handleMarkerMouseLeave = () => {
+    // Flag the tooltip to close after a timeout, which could be unflagged by another event (entering a marker or tooltip).
+    this.setState({ tooltipShouldClose: true });
 
-  closePopup = popupInfo => {
-    this.setState({
-      popups: reject({ index: popupInfo.index }, this.state.popups)
-    });
+    const delayTimeMs = 2000;
```

Comment:
Maybe move this to the top as a constant?

Also, is 2000 too long? Zillow seems to be between 500 and 1000ms.

Reply:
👍 it did seem a little long. shortened to 1000ms